README.md

```diff
@@ -21,3 +21,7 @@ http://nbviewer.ipython.org/github/SciTools/courses/blob/master/course_content/n
 ### An introduction to Iris
 6 hours &mdash; depends on "Cartopy in a nutshell"
 http://nbviewer.ipython.org/github/SciTools/courses/blob/master/course_content/notebooks/iris_intro.ipynb?create=1
+
+### An introduction to Dask
+3 hours &mdash; depends on "An introduction to Iris"
+http://nbviewer.jupyter.org/github/SciTools/courses/blob/dask_course/course_content/notebooks/dask_intro.ipynb
```

Comment:
Note that this link will need to be updated to point to `master` when this branch gets merged into master.

Reply:
I have made a note on the JIRA ticket to remind us to do this.

Reply:
Nice. I assume that this PR can only be merged once the content it there, right?

Also, the naked URL is kinda ugly ... can't we use a markdown alias instead?

Reply:
@bjlittle the content is already there! The pattern used here for the URL is the same as the rest of the README... so if we change one we should change them all. I've never really been that bothered about it.

Reply:
Ah yup, just got the `dask_course` part of the link, thanks.

I'll make the changes later to all the links ... whilst we're here, let's polish as we go :stuck_out_tongue_winking_eye: